roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -74,6 +74,9 @@ kubernetesMasterConfig:
   masterCount: 1
   masterIP: ""
   podEvictionTimeout: ""
+  proxyClientInfo:
```

Comment:
does this need to be conditionalized for version as well? it'll get ignored pre-3.1, but just for hygiene...